lib/openId.js

```diff
@@ -614,9 +614,15 @@ async function _getAuthorizationRequest({req}) {
         } = step.openId || {};
         if(client_id) {
           authorizationRequest.client_id = client_id;
+        } else {
+          authorizationRequest.client_id =
+            `${exchanger.id}/exchanges/${exchange.id}` +
+            '/openid/client/authorization/response';
         }
         if(client_id_scheme) {
           authorizationRequest.client_id_scheme = client_id_scheme;
+        } else if(authorizationRequest.client_id_scheme === 'undefined') {
```

Comment:
Should this be checking for `'undefined'` or `undefined` like in the `nonce` check below?